app/src/main/java/org/dhis2/usescases/main/MainPresenter.kt

```diff
@@ -257,4 +260,13 @@ class MainPresenter(
             Constants.DATA
         )
     }
+
+    fun wasSyncAlreadyDone(): Boolean {
+        return syncIsPerformedInteractor.execute()
+    }
+
+    fun onDataSuccess() {
+        userManager.d2.dataStoreModule().localDataStore().value(WAS_INITIAL_SYNC_DONE)
+            .blockingSet("True")
```

Comment:
Should we move the hardcoded string to a const value in a LocalDataStoreKey class or something like that. This way we could avoid "true" != "True" in case we don't know exactly the value.

Reply:
Yes let me change it
Thanks @Balcan